doesn't compile with current go master

```
# github.com/sourcegraph/go-langserver/langserver/internal/godef/go/parser
../../sourcegraph/go-langserver/langserver/internal/godef/go/parser/parser.go:1213:46: too few values in &ast.CompositeLit literal
```

--- a/langserver/internal/godef/go/parser/parser.go
+++ b/langserver/internal/godef/go/parser/parser.go
@@ -1210,7 +1210,12 @@ func (p *parser) parseLiteralValue(typ ast.Expr) ast.Expr {
 	}
 	p.exprLev--
 	rbrace := p.expect(token.RBRACE)
-	return &ast.CompositeLit{typ, lbrace, elts, rbrace}
+	return &ast.CompositeLit{
+		Type:   typ,
+		Lbrace: lbrace,
+		Elts:   elts,
+		Rbrace: rbrace,
+	}
 }
 
 // checkExpr checks that x is an expression (and not a type).

